src/media/js/addon/reducers/submit.js

```diff
@@ -15,7 +15,7 @@ const initialState = {
   message: '',
   uploadLoaded: null,
   uploadTotal: null,
-  validationErrorMessage: '',
+  validationError: null,
```

Comment:
Either work, don't see a real practical difference. When set by the server it will always look like `{key: value, message: value}`